extensions/positron-r/syntaxes/r.tmGrammar.src.json

```
@@ -455,7 +455,7 @@
 			]
 		},
 		"roxygen-example": {
-			"begin": "(^\\s*#+')\\s*(@examples)\\s*$",
+			"begin": "(^\\s*#+')\\s*(?:(@examples)\\s*|(@examplesIf)\\s.*)$",
```

@examples is now non-capturing.

Should this be:

```suggestion
			"begin": "(^\\s*#+')\\s*(@examples|@examplesIf)\\s*$",
```